CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing
+
+## self-service-reasons
+
+### Developing
+
+First clone this repository and create a new branch from master where you're going to make all the changes.
+
+#### Add Translations
+
+To add a new language file you'll only need to find the lastest version and add the desired file to the version folder.
+
+#### Fix Translations
+
+To fix any messages you'll need to copy the whole version folder along with its messages and rename it as the next version (folowing [Semantic Versioning](https://semver.org/)). Then, make the desired changes in the new version folder.
+
+### Release
+
+Once you're finished with your changes and your PR has already been approved by at least 2 developers, you may run `releasy [minor/patch] --stable`. This will trigger one of our webhook to Pachamama, which is our build tool for this project. After that, you should merge the branch to master.
```

Comment:
Not sure, but I think this project is not built by Pachamama. Also, I would add a link to [releasy](https://github.com/vtex/releasy)